packages/web-reporter/writeReport.ts

```diff
@@ -1,11 +1,44 @@
 import fs from "fs";
+import { TestCaseResult } from "@perf-profiler/types";
+
+// We should have better management of the time interval over the repo
+const MEASURE_INTERVAL = 500;
+
+export const getMeasuresForTimeInterval = ({
+  duration,
+  skip,
+  results,
+}: {
+  duration: number | null;
+  skip: number;
+  results: TestCaseResult[];
+}): TestCaseResult[] => {
+  const firstMeasureIndex = skip / MEASURE_INTERVAL;
+
+  return duration
+    ? results.map((result) => ({
+        ...result,
+        iterations: result.iterations.map((iteration) => ({
+          ...iteration,
+          measures: iteration.measures.slice(
+            firstMeasureIndex,
+            firstMeasureIndex + duration / MEASURE_INTERVAL + 1
```

Comment:
If duration === 300, the first index will be firstIndex + 1.6.

We should use Math.ceil on the result